test/SynthsSdk.test.ts

```diff
@@ -49,9 +53,26 @@ describe("Synths SDKs", function () {
       upunksAsset = synthsSDK.connectAsset("upunks-0921");
     });
 
-    // @todo Add tests
+    // @todo Add tests.
     describe("Interact with asset", function () {
+      it("helpers - success", async function () {
+        this.timeout(100000);
+        console.log(await getYamSynthsTotalTVL());
```

Comment:
add expect, with more details to what are we expecting here and for the console log you can keep it but comment it